crypto/src/context.rs

```diff
@@ -88,22 +88,26 @@ impl CentralContext {
         }
     }
 
-    pub(crate) async fn callbacks(
-        &self,
-    ) -> CryptoResult<RwLockReadGuardArc<Option<Arc<dyn CoreCryptoCallbacks + 'static>>>> {
+    // This is going to be needed soon.
+    #[expect(dead_code)]
```

Comment:
Is this still dead code?

Reply:
Yep, otherwise this would throw an error :) (that's the difference to `#[allow()]`)